src/modules/customfields/customfields.service.ts

```
@@ -1,450 +1,274 @@
+          throw new BadRequestException(`Field ${field.label} must be a valid date`);
+        }
+        break;
+
+      case FieldType.DATETIME:
+        if (isNaN(Date.parse(value))) {
+          throw new BadRequestException(`Field ${field.label} must be a valid datetime`);
+        }
+        break;
+
+      case FieldType.EMAIL:
+        const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)